src/service/poc_mobile.proto

```diff
@@ -37,6 +43,12 @@ message cell_heartbeat_req_v1 {
 
 message cell_heartbeat_resp_v1 { string id = 1; }
 
+//cell heartbeat report outputted by ingestor, tagged with received_timestamp
```

Comment:
Run cargo fmt.. or enable format on save in your editor

Reply:
Ran `cargo fmt` on the project and it is not showing any changes, but still failing

Reply:
oh my bad.. cargo fmt is for rust code.. you need to install clang-format and run the proto formatter that it supports.  vscode has a plugin that does this work for you if you use that